src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,33 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+import codeu.chat.util.Uuid;
+import codeu.chat.util.Time;
+
+public final class ServerInfo {
+	private final static String SERVER_VERSION = "1.0.0";
```

Comment:
The indentation inside this class is kinda wild.

As suggested in https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation, please just use two spaces to indent each subsequent level throughout this file. @mlk209